cmd/commands/common.go

```diff
@@ -265,6 +267,77 @@ func promptSummaryToUser(ctx context.Context, finalParameters map[string]string,
 	return false, nil
 }
 
+func getKubeContextNameFromUserSelect(cmd *cobra.Command, kubeContextName *string) error {
+	if !store.Get().Silent {
+		configAccess := clientcmd.NewDefaultPathOptions()
+		conf, err := configAccess.GetStartingConfig()
+		if err != nil {
+			return err
+		}
+
+		contextsList := conf.Contexts
+		currentContext := conf.CurrentContext
+		var contextsNames []string
+
+		for key := range contextsList {
+			if key == currentContext {
+				key = key + " (current)"
+			}
+			contextsNames = append(contextsNames, key)
+		}
+
+		templates := &promptui.SelectTemplates{
+			Selected: "{{ . | yellow }} ",
+		}
+
+		labelStr := fmt.Sprintf("%vSelect kube context%v", CYAN, COLOR_RESET)
+
+		prompt := promptui.Select{
+			Label:     labelStr,
+			Items:     contextsNames,
+			Templates: templates,
+		}
+
+		_, result, err := prompt.Run()
+		if err != nil {
+			return fmt.Errorf("Prompt error: %w", err)
+		}
+
+		match, err := regexp.MatchString(`.+ \(current\)`, result)
```

Comment:
Should we protect against the unlikely case somebody has a context he named "my (current)"?

Reply:
Maybe we should just compare the `result` with `currentContext + " (current)"`, and if they are equal, just replace `result` with `currentContext`?
I'm not sure it will solve all edge cases, but it is a simpler way to do it.

Reply:
What is the 1st return value of the `prompt.Run()`? If it's the selected index, that would be the simplest way to continue, wouldn't it?